### DISCUSSION TITLE
Question about running a program

### DISCUSSION THREAD

**Discussion by Johnney12138** [idaholab/moose#22992] [Q&A General] [answered]
Hello,
I'm new to moose. I ran into problems while running the program. I received the following error message. How to solve this problem？Thanks!
```
Linking Executable /Users/liunan/projects/article3/stork-opt...
Undefined symbols for architecture x86_64:
  "StorkTestApp::registerApps()", referenced from:
      _main in main.x86_64-apple-darwin13.4.0.opt.o
ld: symbol(s) not found for architecture x86_64
clang-12: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [/Users/liunan/projects/moose/framework/app.mk:416: /Users/liunan/projects/article3/stork-opt] Error 1
```

**Comment by GiudGiud** [comment]
Hello

Did you modify the files after creating a new app?
This looks like the includes are present in the folder, but some of the source got deleted

Did you call your app stork?

Guillaume

**Comment by Johnney12138** [comment]
Hi @GiudGiud ,
Thank you for your reply.
The process of creating an application is attached as follows. I wonder whether this process causes some of the sources to get deleted. 
All the best,
Nan Liu
![error](https://user-images.githubusercontent.com/71909674/208595802-866f0d82-9728-4a29-ac62-6d14d5e6fbbd.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello
'
You are right the process definitely failed.
Can you try running it from the projects folder

so:
./moose/scripts/stork.sh new_cat

Guillaume

**Reply by Johnney12138** [reply]
Hello @GiudGiud ,
Thank you for your response.
Running "./moose/scripts/stork.sh new_cat" from the projects folder doesn't work. I have no idea why this error is appearing.

Best,
Nan Liu

<img width="584" alt="error" src="https://user-images.githubusercontent.com/71909674/208821483-1d99a1bc-7079-463c-8e87-370381271bee.png">

**Reply by GiudGiud** [reply]
The script says why. 
projects is a git directory. That s not normal

you should wipe the directory and try again

**Reply by milljm** [reply]
Please do not post pictures of Text as well. Doing so makes it _impossible_ to use the search feature. You started off just fine in your original post :) Not sure what made you switch to pictures!
